### PR TITLE
Add TLC

### PR DESCRIPTION
**Pull request by chameleonTK**
Added TLC dataset

### CODE REVIEW

File: datasets/tlc/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+annotations_creators:
+- expert-generated
```
Comment (lhoestq) [inline datasets/tlc/README.md:3]:
```suggestion
```

File: datasets/tlc/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+BibTeX:
+```
+@misc{
+  author={Sawatphol, Jitkapat},
+  title={Thai Literature Corpora},
+  year={2019},
+  howpublished={\\url{https://attapol.github.io/tlc.html}}
+}
+```
```
Comment (lhoestq) [inline datasets/tlc/README.md:114]: looks like you removed many paragraph titles from the dataset card.
Could you re-add them like in the template ? you can leave their content with `[More Information Needed]`

**Review by lhoestq — COMMENTED** [review]
Awesome ! Thanks for adding this dataset :)

I left a few comments

Also it looks like the dummy_data.zip files are quite heavy (2MB). Could you try to reduce the size to a few KB if possible ?

To do so feel free to take a look at the text files inside and remove unnecessary chunks of text.

**Comment by chameleonTK** [comment]
I have reduced the size of the dummy file and added README sections as you suggested.

**Comment by chameleonTK** [comment]
I have a little issue to run the test. It seems there is no failed case in my machine.

**Comment by lhoestq** [comment]
Thanks !
It looks like the PR includes changes to many other files than the ones of `tlc`, can you create another branch and another PR ?

**Pull request closed without merging**